src/store/login/index.ts

```diff
@@ -37,6 +39,7 @@ export const initialState: LoginState = {
 
 export const loginByEmail = createAction<{ email: string; password: string }>(SagaActionTypes.EmailLogin);
 export const loginByWeb3 = createAction<Connectors>(SagaActionTypes.Web3Login);
+export const switchLoginStage = createAction<LoginStage>(SagaActionTypes.SwitchLoginStage);
```

Comment:
Wasn't sure if we'd want to limit the params to `Web3 | Email`, so it's not possible to switch to `Done`, but unsure.